chapters/file-tags.md

```diff
@@ -50,10 +50,62 @@ Multiple-line values are not recommended, because doing so will make it harder f
 
 Version 3.0 of the [REUSE Software guidelines](https://reuse.software/spec/) implements this format, via a recommendation to use the tag `SPDX-FileCopyrightText:` to include copyright notices as part of a file's comment headers.
 
-## H.3 Caveats <a name="H.3"></a>
+## H.3 File Tags Within Snippets <a name="H.3"></a>
+
+If certain SPDX file tags are to apply only to a certain snippet instead of the whole file, the start of such snippet should be marked with `SPDX-SnippetBegin` and its end with `SPDX-SnippetBegin`.
+
+Any Snippet Information (Clause [9](snippet-information.md)) and short-form license identifiers (Annex [E](using-SPDX-short-identifiers-in-source-files.md)) tags found between begin and end tags mentioned above apply only to such snippet.
+
+Examples:
+
+Simple stand-alone example:
+
+```text
+SPDX-SnippetBegin
+SPDX-License-Identifier: MIT
+SPDX-SnippetCopyrightText: 2022 Jane Doe <jane@example.com>
+
+...
+
+SPDX-SnippetEnd
+```
+
+Two snippets with a different license and additional information in the broader context of a file:
+
+```text
+SPDX-License-Identifier: GPL-2.0-or-later
+SPDX-FileCopyrightText: Copyright contributors to the Foo project.
+
+...
+
+SPDX-SnippetBegin
+SPDX-License-Identifier: MIT
+SPDX-SnippetCopyrightText: 2022 Jane Doe <jane@example.com>
+SDPX—SnippetName: functionX from project Bar
+SPDX-SnippetComment: A complex function X that was copy-pasted from project Bar
+
+...
+
+SPDX-SnippetEnd
+
+...
+
+SPDX-SnippetBegin
+SPDX-License-Identifier: BSD-2-Clause
+SPDX-SnippetCopyrightText: Copyright Example Company
+
+...
+
+SPDX-SnippetEnd
+
+```
+
```

Comment:
Add something like:

Snippets may nest, and this is denoted by having `SPDX-SnippetBegin`/`SPDX-SnippetEnd` pairs within other pairs, in the same way that parentheses nest in mathematical expressions.
In the case of nested snippets, the SPDX file tags are considered to apply to the inner-most snippet.